### ISSUE TITLE
Accessibility - Support changing the system font size

### ISSUE THREAD

**Issue by Ross-Gibson** [Triggertrap/triggertrap-ios#52]
The font size throughout the app should respond to the users preferred size within accessibility settings: **Settings -> General -> Accessibility-> Larger Text**.

We've had a few support tickets related to this issue.

**Comment by alex-taffe** [comment]
I can get this one

**Comment by alex-taffe** [comment]
@Ross-Gibson I'm going to wait to do this one until you've merged #48, #49, and #50 because I can nearly guarantee it's going to cause a nightmare of merge conflicts

**Comment by Ross-Gibson** [comment]
@alex-taffe yeah that makes sense. I've reviewed two of the outstanding PR's and one now has conflicts. If you're able to resolve those, I will review the changes again, then you can move on to this task.

Sorry again for the slowness lately - new born baby makes developing a bit harder!

**Comment by alex-taffe** [comment]
@Ross-Gibson got it

**Comment by alex-taffe** [comment]
Starting on this now
